src/actions/selected.js

```diff
@@ -75,18 +70,14 @@ export const acClearSelectedItemActiveTypes = () => ({
 export const tSetSelectedDashboardById = id => async (dispatch, getState) => {
     dispatch(acSetSelectedIsLoading(true))
 
-    const snackbarTimeout = setTimeout(() => {
-        const dashboardName = (
-            sGetDashboardById(getState(), id) || EMPTY_DASHBOARD
-        ).displayName
-        if (sGetSelectedIsLoading(getState()) && dashboardName) {
-            loadingDashboardMsg.name = dashboardName
+    const alertTimeout = setTimeout(() => {
+        const name = sGetDashboardById(getState(), id)?.displayName
 
+        if (sGetSelectedIsLoading(getState()) && name) {
             dispatch(
-                acReceivedSnackbarMessage({
-                    message: loadingDashboardMsg,
-                    open: true,
-                })
+                acSetAlertMessage(
+                    i18n.t(`Loading dashboard – ${name}`, { name })
```

Comment:
🤔  this doesn't look is using the correct syntax:
https://www.i18next.com/translation-function/interpolation

I suspect is not doing what we want?
```suggestion
                    i18n.t("Loading dashboard – {{name}}", { name })
```

Reply:
Oops, you're right 😅